### DISCUSSION TITLE
Privacy and Legal

### DISCUSSION THREAD

**Discussion by genuineswe** [cline/cline#411] [Feature Requests]
I wanna ask about privacy and legal data history and how safe it is?

**Comment by nickbaumann98** [comment]
Cline itself does not track or store any of your data. The LLM provider you use may or may not.

**Comment by WarrenMfg** [comment]
Here is their privacy policy: https://github.com/cline/cline/blob/main/docs/PRIVACY.md

Hope that helps!
